.ci_support/linux_64_.yaml

```diff
@@ -1,8 +1,10 @@
 cdt_name:
-- cos7
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
Note this bumps the Python min to 3.9, which is [the oldest Python version supported in conda-forge]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/e1861a47bc93e952bfe69a7bbdbce78220b8369c/recipe/conda_build_config.yaml#L786-L801 )